src/MiniCover/Reports/CloverReport.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using MiniCover.Model;
+
+namespace MiniCover.Reports
+{
+    public static class CloverReport
+    {
+        public static void Execute(InstrumentationResult result, string output, float threshold)
+        {
+            var hits = File.Exists(result.HitsFile)
+                ? File.ReadAllLines(result.HitsFile)
+                    .Select(h => int.Parse(h))
+                    .GroupBy(h => h)
+                    .ToDictionary(h => h.Key, h => h.Count())
+                : new Dictionary<int, int>();
+
+            var document = new XDocument(
+                new XDeclaration("1.0", "utf-8", null),
+                CreateCoverageElement(result, hits)
+            );
+
+            var xmlWriterSettings = new XmlWriterSettings
+            {
+                Indent = true
+            };
+
+            Directory.CreateDirectory(Path.GetDirectoryName(output));
+
+            using (StreamWriter sw = File.CreateText(output))
+            using (XmlWriter writer = XmlWriter.Create(sw, xmlWriterSettings))
+            {
+                document.WriteTo(writer);
+            }
+        }
+
+        private static XElement CreateCoverageElement(InstrumentationResult result, IDictionary<int, int> hits)
+        {
+            var timestamp = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+
+            return new XElement(
```

Comment:
It would be easiest to understand when we would be working on an object. Saving to XML should be with using serialization to XML. Manual creation of every node is not the best solution.

Reply:
I don't have any preference between Linq to XML or objects serialization.
Other reports already use Linq to XML, so maybe is better to keep using it.

Reply:
@lucaslorentz I just prepare PR for changing linq to xml into object serialization. I understand you are not interested to?

Reply:
Can you create a separate PR with it? I would like to see it.

Since it's not a must have for clover report, I'm going to merge this one as it is.